src/types/ws.ts

```diff
@@ -19,10 +20,19 @@ export interface QuestCompletedEvent {
     };
 }
 
+export interface CrewMember {
+    imageUrl: string | null;
```

Comment:
See the comment at the other PR: many of these information do not change over time, so it would be worth asking them down only once and not every 15 secs